### PR TITLE
✨ Adding Pressboard implementation of amp-ad

### PR DESCRIPTION
**Pull request by omar84**
Adding Pressboard implementation of amp-ad to the amphtml project

### CODE REVIEW

**Comment by omar84** [comment]
@zhouyx this is the pull request for the amp-ad changes, can you please take a look?

**Comment by omar84** [comment]
@zhouyx sorry about that, the changes were not live on our server, now they are. can you check again please? thanks.

File: ads/pressboard.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (zhouyx) [inline ads/pressboard.js]: nit: 2018

File: ads/pressboard.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (zhouyx) [inline ads/pressboard.md]: nit: 2018

File: validator/testdata/feature_tests/ads.out [hunk 3]
```diff
@@ -166,6 +166,7 @@ PASS
 |          <option>plista</option>
 |          <option>polymorphicads</option>
 |          <option>popin</option>
+|          <option>pressboard</option>
```
Comment (zhouyx) [inline validator/testdata/feature_tests/ads.out]: same here. No need to change this file.

File: ads/_config.js [hunk 4]
```diff
@@ -727,6 +727,10 @@ export const adConfig = {
 
   'postquare': {},
 
+  'pressboard': {
+    renderStartImplemented: true,
```
Comment (zhouyx) [inline ads/_config.js]: I couldn't see the `render-start` signal when I test locally, could you double check that the renderStart API is called correctly? Thank you.

**Comment by omar84** [comment]
@zhouyx All your comments have been addressed, thank you.

File: validator/testdata/feature_tests/ads.out (hunk 3 above)
Comment (omar84) [reply validator/testdata/feature_tests/ads.out]: this change was added to pass the tests for the merge

**Comment by omar84** [comment]
@zhouyx your comments have been addressed, and all checks have passed. can you please check again? thanks.

File: ads/_config.js [hunk 5]
```diff
@@ -727,6 +727,11 @@ export const adConfig = {
 
   'postquare': {},
 
+  'pressboard': {
+    // prefetch: 'https://adserver.pressboard.ca/js/amp-ad.js',
```
Comment (zhouyx) [inline ads/_config.js]: Why comment out? Do we want them?

Please also consider adopting the `renderStartAPI` to improve user experience. More information can be found [here](https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis). Please let us know if there's any question. Thanks

File: validator/testdata/feature_tests/ads.out (hunk 3 above)
Comment (zhouyx) [reply validator/testdata/feature_tests/ads.out]: It's not required if you don't change validator/testdata/feature_tests/ads.html file.

File: ads/_config.js (hunk 5 above)
Comment (omar84) [reply ads/_config.js]: implemented renderStart, removed comments

**Comment by zhouyx** [comment]
@omar84 LGTM. Saucelab is down, let's wait and see.

**Comment by omar84** [comment]
@zhouyx  can I do something? can we rerun the checks?

**Comment by zhouyx** [comment]
I restarted the build.

**Comment by omar84** [comment]
@zhouyx thank you for rebuilding!, the build seem to have passed.

**Comment by zhouyx** [comment]
Thank you @omar84 PR merged.

**Comment by omar84** [comment]
@zhouyx all looks good, can we expect a merge soon? thank you.

**Comment by omar84** [comment]
@zhouyx thank you very much! when can we start using it? is it live yet?

**Pull request merged**